### PR TITLE
Usman/lp 1504

### PR DESCRIPTION
**Pull request by emraan004**

### CODE REVIEW

File: common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                                        if block.block_type == 'openassessment':
+
+                                            # check if this chapter is 2 weeks older or not.
+                                            module_access_days = delta_days.days - (index_chapter * 7)
+
+                                            try:
+                                                anonymous_user = AnonymousUserId.objects.get(user=user,
+                                                                                             course_id=course.id)
+                                            except:
+                                                continue
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py]: Please add logs here

File: common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        courses = CourseOverview.objects.filter(self_paced=True)
+
+        for course in courses:
+            enrollments = CourseEnrollment.objects.filter(course_id=course.id, is_active=True)
+
+            for enrollment in enrollments:
+
+                today = datetime.now(utc).date()
+                delta_days = today - enrollment.created.date()
+                current_module = delta_days.days / 7 + 1
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py]: let's make this division clearer

File: common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                                                    student_item__item_id=block)
+                                                log.info('Response Created at: %s',
+                                                         response_submissions.created_at.date())
+
+                                                response_submission_delta = today - response_submissions.created_at.date()
+
+                                                if module_access_days >= 14 and response_submission_delta.days >= 14:
+                                                    try:
+                                                        AssessmentWorkflow.objects.get(
+                                                            status=AssessmentWorkflow.STATUSES[0],
```
Comment (emraan004) [inline common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py]: please comment what this status[0] is?

File: common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                                for index_vertical, vertical in enumerate(verticals.children):
+                                    vertical_blocks = modulestore().get_item(vertical)
+                                    for index_block, block in enumerate(vertical_blocks.children):
+                                        if block.block_type == 'openassessment':
+
+                                            try:
+                                                anonymous_user = AnonymousUserId.objects.get(user=user,
+                                                                                             course_id=course.id)
+                                            except AnonymousUserId.DoesNotExist:
+                                                log.info('Anonymous Id doesn\'t exists for User: %s', user)
```
Comment (ahmed-faraz46) [inline common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py]: `log error` please

File: common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+                                                continue
+                                            except AnonymousUserId.MultipleObjectsReturned:
+                                                log.info('Multiple Anonymous Ids for User: %s', user)
```
Comment (ahmed-faraz46) [inline common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py]: `log error` please

**Comment by muak50** [comment]
Change will be catered in next iteration.

**Pull request merged**